src/client/graphql/AddCampaignCode.graphql

```diff
@@ -1,8 +1,19 @@
-mutation AddCampaignCode($id: ID!, $code: String!) {
+mutation AddCampaignCode($id: ID!, $code: String!, $locale: Locale!) {
   quoteCart_addCampaign(id: $id, code: $code) {
     ... on QuoteCart {
       id
+      bundle {
```

Comment:
This addition produces the following warning while adding a new campaign code:

<img width="1570" alt="Screenshot 2022-02-15 at 10 56 00" src="https://user-images.githubusercontent.com/19200662/154038086-17cdfbc4-e1e4-47d8-aac2-6cf2e5902d51.png">

Although it's not a current problem for us because a `QuoteCart` query is automatic triggered after `AddCampaignCode` mutation, it might be a problem in the future. The root of these problem is that `QuoteBundle`  type is not identifiable so every time we update this type we might be erasing some of the apollo cache. Not sure if we want to do something about it at this moment... 🤔

Reply:
Ahhh.... very tricky